ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst

```diff
@@ -0,0 +1,163 @@
+.. Copyright (C)  Google, Runestone Interactive LLC
+   This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+   International License. To view a copy of this license, visit
+   http://creativecommons.org/licenses/by-sa/4.0/.
+
+Motivating Scatterplots
+=======================
+
+Weather is something that changes based on location. Cities closer to the north
+and south pole are generally pretty cold, while cities close to the equator are
+generally pretty warm. But what if you wanted to quantify this relationship in
+order to make specific predictions? Instead of saying “cities are generally
+colder closer to the north pole,'' you could say, “as latitude increases by 10
+degrees, temperature decreases by 20 degrees.”
+
+Given the following dataset with latitude, longitude, and weather data for
+different cities in the United States, you can use histograms to plot latitudes
+and temperatures independently as shown below.
+
+
+.. image:: figures/lat_temp_histograms.png
+   :alt: Two histograms side by side. One is of of latitudes of major cities in the US, the other is of temperatures in January.
+
+Notice that this shows you the distribution of each variable alone, but tells
+you nothing about the overall relationship between latitude and temperature.
+How can you investigate the relationship with the tools you have?
+
+Since you know that temperature is related to location, you could group the
+cities based on their latitude, and then compare the mean temperature for each
+group. Comparing the means for each group will quantify if and how much latitude
+impacts temperature.
+
+A latitude closer to 0 degrees means a city is closer to the equator, and a
+latitude closer to 45 degrees means a city is closer to the poles. For cities in
+the northern hemisphere, like those in the USA, smaller latitudes indicate
+southern cities, and larger latitudes indicate northern cities. See here for a
+map of the USA with latitude and longitude lines.
+
+You can split up cities in the United States into three regions, “Northern,”
+“Southern,” and Central to more easily analyze the data. Most cities in the
+continental states are between 25 and 50 degrees, so to group these, you can
+label cities south of 35 degrees latitude “Southern”, cities north of 40 degrees
```

Comment:
Throughout this file, you need to be consistent as to whether you put the commas and periods within the quotes or not. It doesn't matter too much which you choose (there are some examples throughout this whole file), but be consistent.